modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java

```diff
@@ -65,30 +87,29 @@ protected static String addNewImportsJSPHeader(
 
 			return StringUtil.replaceFirst(
 				newContent, jspHeader,
-				getNewImportsJSPHeader(new String[] {jspHeader}, newImports));
+				getNewImportsJSPHeader(jspHeader, newImports));
 		}
 
-		return getNewImportsJSPHeader(new String[0], newImports) + newContent;
+		return getNewImportsJSPHeader(null, newImports) + newContent;
 	}
 
 	protected static String getNewImportsJSPHeader(
-		String[] jspHeaders, String[] newImports) {
+		String lastJSPHeader, String[] newImports) {
 
 		StringBundler sb = new StringBundler(4);
 
-		for (String jspHeader : jspHeaders) {
-			sb.append(jspHeader);
+		if (lastJSPHeader != null) {
```

Comment:
@nicolas-sss, can we use an empty string instead of `null`?

I want to prevent a `NullPointerException` as the one we handled recently in the method `getVariableName`.